openzaak/src/main/kotlin/com/ritense/openzaak/liquibase/changelog/ChangeLog20211130.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2015-2020 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.openzaak.liquibase.changelog
+
+import com.fasterxml.jackson.databind.node.ArrayNode
+import com.fasterxml.jackson.databind.node.ObjectNode
+import com.fasterxml.jackson.databind.node.TextNode
+import com.ritense.document.domain.impl.Mapper
+import liquibase.change.custom.CustomTaskChange
+import liquibase.database.Database
+import liquibase.database.jvm.JdbcConnection
+import liquibase.exception.CustomChangeException
+import liquibase.exception.ValidationErrors
+import liquibase.resource.ResourceAccessor
+
+class ChangeLog20211130 : CustomTaskChange {
```

Comment:
Ik zou toch deze classname baseren op wat de class doet, niet wat de id van de changelog is.

Reply:
Bij change-logs is het wel handig als er een volgorde inzit. Vandaar die datum. Heb er nu dit van gemaakt:
ChangeLog20211130AddProcessDefinitionKeyToServiceTaskHandler
:)

De changelog zal nooit veranderen en nooit op andere plekken gebruikt mag worden. Dus ik denk dat de naam niet heel belangrijk is.